confirm-deposit.md

```diff
@@ -0,0 +1,120 @@
+# Confirm Deposits to the Exchange
+
+Deposits are made in several stages. 
+After the user deposits funds to their assigned reception wallet (step 1 in [Detecting Deposits](https://docs.flare.network/exchange/architecture/#detecting-deposits)), the [Exchange server](https://docs.flare.network/exchange/architecture/#architecture-of-an-exchange) monitors for deposits and updates the Balances database. 
+Run the code below to have the Exchange server listen for deposit transactions and confirm the deposits.
+
+On a [blockchain](https://docs.flare.network/tech/glossary/), multiple [validators](https://docs.flare.network/tech/validators/) examine newly created transactions and decide on whether they are valid and which [block](https://docs.flare.network/tech/glossary/) to add to the chain next. 
+It takes time for all the validators to agree. 
+Until they reach agreement, validators may add or revert transactions to a block. 
+After the validation process is complete, reversing a transaction is very unlikely. 
+To avoid the risk of confirming a block that could be reverted, confirm deposits several blocks back (for example, five blocks back as in the code below). 
+Older blocks are less likely to be reverted.
+
+Newly submitted deposits to the receiving address are labeled "pending."
+After five more blocks have been created, valid deposits are labeled "confirmed."
+
+Run the code below to see pending and confirmed transactions and which block number they were added to.
+
+## One-Time Setup
+
+1. Install [node.js](https://nodejs.org/en/download/). 
+
+   On the command line, run `node -v`  to check your installation.
+
+2. Clone the Flare repository: 
+
+   ```
+   git clone https://github.com/flare-foundation/flare.git
+   cd flare
+   ```
+
+   See [the Flare repository on GitHub](https://github.com/flare-foundation/flare).
+
+3. Install the web3.js library: 
+
+   `npm install web3`
```

Comment:
Remember to use single backquotes only in inline text. This is a standalone code block so it should use triple backquotes (as in the `git clone` block above).

Reply:
That makes sense! (My former reviewers have let me get away with this!) Much better.